documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+[NOTICE](#NOTICE)
+[Template](#Template)
+[Comments](#Comments)
+
+## DEVNOTES
+
+feature/184 - version 0.2 documentation
+Working on formatting this document like the [spawnPatterns readme.](openscope/src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md).
+Plans:
+    - Remove all "// <explanation>" lines from the template so it can be copy/pasted.
```

Comment:
I don't think that this should be the case. I think it should just be a file that explains everything as it already is, it's an airport-format markdown file which is for explanation - an editable version can be already found from all the airports we already have.